DatadogInternal/Sources/NetworkInstrumentation/URLSession/URLSessionTaskDelegateSwizzler.swift

```diff
@@ -0,0 +1,115 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+internal func key(from anyClass: AnyClass) -> String {
+    String(reflecting: type(of: anyClass))
+}
```

Comment:
suggestion/ Too generic for `internal` namespace. Think of nesting or specialising.

Reply:
fair call, moved.